src/enzyme_ad/jax/Dialect/EnzymeXLAOps.td

```diff
@@ -189,4 +189,22 @@ def CommRegionOp : EnzymeXLA_Op<"comm_region", [
   );
 }
 
+def LUFactorizationOp: EnzymeXLA_Op<"lu_factorization", [Pure]> {
```

Comment:
small proposal: this feels like a small subdialect inside `enxymexla`, so how about calling  the op `enzymexla.linalg.lu`? i know they do this in the IREE dialects